stages/vault/sharedsvc/vault-policies/lead-aws-developer.hcl

```diff
@@ -13,3 +19,9 @@ path "lead/metadata" {
 path "lead/metadata/aws" {
     capabilities = ["read", "list"]
 }
+
+// create child tokens
+
+path "auth/token/create" {
+    capabilities = ["update"]
+}
```

Comment:
this was needed for the `Developer` role to login to Vault using the Vault tf provider, as that process creates child tokens.